### PR TITLE
feat: #2877 Create a subscriptions section on the Admin Portal

### PR DESCRIPTION
**Pull request by Bdragon93**
# Pull request checklist

feat: #2877 

**Any additional notes for reviewer (optional):**

### CODE REVIEW

File: packages/admin-portal/src/core/router.tsx [hunk 1]
```diff
@@ -34,6 +35,7 @@ const Router = () => {
                 <PrivateRoute path={Routes.DEV_MANAGEMENT} component={DevsManagementPage} exact fetcher />
                 <PrivateRoute path={Routes.STATS} component={Statistics} exact />
                 <PrivateRoute path={Routes.CUSTOMERS} component={CustomersPage} exact fetcher />
+                <PrivateRoute path={Routes.SUBSCRIPTIONS} component={SubscriptionsPage} exact fetcher />
```
Comment (willmcvay) [inline packages/admin-portal/src/core/router.tsx:38]: Remove `fetcher` here as discussed

File: packages/admin-portal/src/utils/route-dispatcher.ts [hunk 2]
```diff
@@ -25,6 +26,12 @@ const routeDispatcher = async (route: RouteValue, params?: StringMap, search?: s
     case Routes.CUSTOMERS:
       store.dispatch(fetchCustomersList({ queryString: search || '' }))
       break
+    case Routes.SUBSCRIPTIONS:
```
Comment (willmcvay) [inline packages/admin-portal/src/utils/route-dispatcher.ts:29]: Remove this case - it is not required

File: packages/admin-portal/src/actions/subscriptions.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { actionCreator } from '../utils/actions'
+import ActionTypes from '../constants/action-types'
+import { SubscriptionModelPagedResult } from '@reapit/foundations-ts-definitions'
+
+export interface fetchSubscriptionListValues {
+  page: number
+  queryString: string
+}
+
+export const fetchSubscriptionList = actionCreator<fetchSubscriptionListValues>(ActionTypes.FETCH_SUBCRIPTION_LIST)
```
Comment (willmcvay) [inline packages/admin-portal/src/actions/subscriptions.ts:10]: Just dispatch this action when you click on a developer in your developer drop down list

**Review by willmcvay — COMMENTED** [review]
As per our Slack chat - otherwise looks good 👍

File: packages/admin-portal/src/components/pages/subscriptions/__tests__/index.test.tsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+      } as SubscriptionsFilterFormValues,
+      {
+        setStatus: jest.fn(),
+      },
+    )
+    expect(mockRouterProps.history.push).toBeCalled()
+  })
+})
+
+// describe('closeDisableMemberModal', () => {
```
Comment (willmcvay) [inline packages/admin-portal/src/components/pages/subscriptions/__tests__/index.test.tsx:101]: I guess will implement when the disabled button works right?

File: packages/admin-portal/src/components/pages/subscriptions/index.tsx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+      Header: '',
+      accessor: 'id',
+      Cell: DeleteButton,
+    },
+  ]
+
+  return (
+    <ErrorBoundary>
+      <Section className="mb-0">
+        <H3>Subscriptions</H3>
```
Comment (willmcvay) [inline packages/admin-portal/src/components/pages/subscriptions/index.tsx:129]: You can just use the H3 not wrapped in a section by using the `isHeadingSection` prop eg `<H3 isHeadingSection>...`

File: packages/elements/src/components/DropdownSelect/index.tsx [hunk 6]
```diff
@@ -88,6 +88,7 @@ export const DropdownSelect: React.FC<DropdownSelectProps> = ({
                     onBlur={handleFieldTouched(form, field)}
                     getPopupContainer={getPopupContainer}
                     dropdownStyle={{ zIndex: 999 }}
+                    // multiple={false}
```
Comment (willmcvay) [inline packages/elements/src/components/DropdownSelect/index.tsx:91]: Think you can remove

**Review by willmcvay — COMMENTED** [review]
Looks good - have not approved yet as I think you have more to do. Happy to merge though, the code looks good, well tested - nice first PR 💯

File: packages/admin-portal/src/components/pages/subscriptions/__tests__/index.test.tsx (hunk 4 above)
Comment (Bdragon93) [reply packages/admin-portal/src/components/pages/subscriptions/__tests__/index.test.tsx:101]: yup

File: packages/admin-portal/src/components/pages/subscriptions/index.tsx (hunk 5 above)
Comment (Bdragon93) [reply packages/admin-portal/src/components/pages/subscriptions/index.tsx:129]: okay, will check it

**Review by willmcvay — APPROVED** [review]
LGTM 👍

**Pull request merged**